packages/web-frontend/package.json

```diff
@@ -12,7 +12,7 @@
   "main": "dist/index.js",
   "scripts": {
     "analyze": "source-map-explorer build/static/js/main.*",
-    "build": "rm -rf dist && run-p \"build:* {@}\" --",
+    "build": "rm -rf dist && run-s \"build:* {@}\" --",
```

Comment:
We are still using `run-p` for most other `build` commands, as they're just building types and source code. Figured it was safe to leave them, but an argument could be made to switch to `run-s` for all I suppose...